core/connection.go

```diff
@@ -214,15 +225,13 @@ func validatePaths(src, dst Chain) error {
 }
 
 func logConnectionStates(src, dst Chain, srcConn, dstConn *conntypes.QueryConnectionResponse) {
-	log.Printf("- [%s]@{%d}conn(%s)-{%s} : [%s]@{%d}conn(%s)-{%s}",
-		src.ChainID(),
-		mustGetHeight(srcConn.ProofHeight),
-		src.Path().ConnectionID,
-		srcConn.Connection.State,
-		dst.ChainID(),
-		mustGetHeight(dstConn.ProofHeight),
-		dst.Path().ConnectionID,
-		dstConn.Connection.State,
+	logger := GetConnectionPairLogger(src, dst)
+	logger.Info(
+		"connection states",
+		"src ProofHeight", mustGetHeight(srcConn.ProofHeight),
+		"src State", srcConn.Connection.State.String(),
+		"dts ProofHeight", mustGetHeight(dstConn.ProofHeight),
+		"dst State", dstConn.Connection.State.String(),
```

Comment:
Fix this in the same way as channel.go.

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/92/commits/05ad46371117929c9ecb2d804aad16f8e5d5194d
Use slog.Group